docs/language/accounts.mdx

```diff
@@ -543,10 +552,14 @@ let nonExistentRef = authAccount.borrow<&{HasCount}>(from: /storage/nonExistent)
 
 ## Storage limit
 
-Accounts storage is limited by its storage capacity.
+An account's storage is limited by its storage capacity.
+
+An account's storage used is the sum of the size of all the data that is stored in an account (in MB).
+An account's storage capacity is a value that is calculated from the amount of FLOW
+that is stored in the account's main FLOW token vault. 
 
-At the end of every transaction all accounts storage used is compared to their storage capacity.
-If for any account its storage used is greater than its storage capacity the transaction fails.
+At the end of every transaction, the storage used is compared to the storage capacity.
+If for any account its storage used is greater than its storage capacity, the transaction fails.
```

Comment:
> If for any account its stora...

Suggested: For all accounts involved in the transaction, if any of the account's storage is greater than its storage capacity, the transaction will fail. 

Reason: Clarity